examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -6125,15 +6125,15 @@
           "enabled": 0,
           "commands": [
             {
-              "name": "LockDoor",
+              "name": "Lock Door",
```

Comment:
:(

Reply:
Matching the spec....  In practice, this does not affect the generated code, particularly.